From 457008cc153ba180a124d1af6d4e7ad574ffa7c4 Mon Sep 17 00:00:00 2001
From: Yiwei Zhang <zzyiwei@chromium.org>
Date: Mon, 18 Jul 2022 05:17:21 +0000
Subject: [PATCH] i965: increase BRW_MAX_UBO to 16

The new limit is legit and it's for virgl to not regress guest desktop
GL version advertised.

Signed-off-by: Yiwei Zhang <zzyiwei@chromium.org>
---
 src/mesa/drivers/dri/i965/brw_context.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mesa/drivers/dri/i965/brw_context.h b/src/mesa/drivers/dri/i965/brw_context.h
index bf00fe01607..ef400ed7793 100644
--- a/src/mesa/drivers/dri/i965/brw_context.h
+++ b/src/mesa/drivers/dri/i965/brw_context.h
@@ -340,7 +340,7 @@ struct brw_program {
 #define BRW_MAX_TEX_UNIT 32
 
 /** Max number of UBOs in a shader */
-#define BRW_MAX_UBO 14
+#define BRW_MAX_UBO 16
 
 /** Max number of SSBOs in a shader */
 #define BRW_MAX_SSBO 12
-- 
2.31.0

